README.md

```diff
@@ -70,11 +70,9 @@ Use the following commands to manage the local lifecycle of animal-rescue
 ./script/local.sh cleanup       # remove the uaa server docker image
 ``` 
 
-Here is a list of special treatment in this sample app to make sure frontend and backend can communicate locally without gateway:
-- Backend has a local profile that adds `spring-security-oauth2-client` to it's dependency, and configure `oauth2Login` in it's security configuration. You can also use `formLogin` or `basicAuth` instead to simplify the setup.
-- After a successful login, the redirect uri is set to `${frontendUrl/rescue/admin}` to mimic the user flow with gateway redirect.
-- `auth` module contains a `uaa` Dockerfile that helps finish the oauth2 login flow. You can skip this step if you use `formLogin` or `basicAuth` for your local security configuration.
-- Frontend sign in link has to point at backend login url in dev mode. 
+#### Local security configuration
+Backend uses Form login for local development with two test accounts - `alice / test` and `bob / test`. 
+Note that in a real deployment with Gateway, OAuth2 login will be managed by the gateway itself, and your app should use `TokenRelay` filter to receive OpenID ID Token in `Authorization` header. See `CloudFoundrySecurityConfiguration` class for an example of Spring Security 5 configuration to handle token relay correctly.
```

Comment:
Could you also add a section about the tag you added for local oauth2 flow example?